stable/kibana/templates/configmap.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "kibana.fullname" . }}
+  labels:
+    app: {{ template "kibana.name" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  config: |
```

Comment:
Should: expose the whole file with its original name at values.yaml. e.g. `kibana.yml`.

Example: https://github.com/kubernetes/charts/blob/master/stable/prometheus/values.yaml#L730

Reply:
Fixed!